zookeeper-jute/src/main/java/org/apache/jute/BinaryInputArchive.java

```diff
@@ -27,8 +27,23 @@
  *
  */
 public class BinaryInputArchive implements InputArchive {
-    static public final String UNREASONBLE_LENGTH= "Unreasonable length = ";
+    public static final String UNREASONBLE_LENGTH= "Unreasonable length = ";
+    public static final int maxBuffer = Integer.getInteger("jute.maxbuffer", 0xfffff);
+    private static final int extraMaxBuffer;
+
+    static {
+        Integer configuredExtraMaxBuffer =
+            Integer.getInteger("zookeeper.jute.maxbuffer.extrasize", maxBuffer);
```

Comment:
one suggestion: existing maxbuffer has a name of "jute.maxbuffer" (note, without zookeeper prefix), so it might be better
that we name this extra size property as "jute.maxbuffer.extrasize" to be consistent with existing style.

also, we need documentation on this new property.

Reply:
There is big problem with jute.maxbuffer. It can not be configured in zoo.cfg.  We are forced to configure it though java system property like -Djute.maxbuffer=someValue.

When we are giving property name zookeeper.jute.maxbuffer.extrasize. it can be configured in zoo.cfg as jute.maxbuffer.extrasize=someValue

This is the reason I do not want to be consistent with jute.maxbuffer.  is it OK?

Reply:
i see - sounds reasonable. I am ok with current name.

side note - I never really remember why we have a few configuration options that's only available through java system properties and what's the criteria of choosing one versus another.